Tina.io - Annual pricing page version 1

Marxoz opened the pull request:
### General Contributing:

- [x] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

<!-- For non content related updates, or fixing things like typos, you can erase the following section -->

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tinacms/tina.io?pullRequest=2325) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/tinacms/tina.io?pullRequest=2325) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: joshbermanssw<br/>:x: Marxoz<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tinacms/tina.io?pullRequest=2325) it.</sub>

Marxoz commented:
Updated Josh's changes to the following:

- Changed switch label 
From
   "Yearly" 
To 
    "Annually"
    
- Changed Annual billing line
From:
   Go yearly and get 2 months free!
To:
   Switch to annual billing and save!
   
This is the version 1 of the page. Betty will be review the page and make more suggestions

bradystroud reviewed (approved):
LGTM
